.github/workflows/publish_nightly.yml

```diff
@@ -39,17 +39,20 @@ jobs:
         with:
           node-version: "18"
           registry-url: "https://registry.npmjs.org"
-          cache: "npm"
+          cache: "pnpm"
           cache-dependency-path: ./typescript
 
+      - name: Install pnpm
+        run: npm install -g pnpm
```

Comment:
Should we use the pnpm setup action?

Reply:
Yes. I did a repo-wide search for `npm install -g pnpm` and thought I got all the references... will go update in any workflows I missed